### DISCUSSION TITLE
value of input tags is hidden

### DISCUSSION THREAD

**Discussion by vhdm** [livewire/livewire#4584] [1. Help] [answered]
Hi there, i have  3 input tag in my form, but values of input tag is hidden :-|
![image](https://user-images.githubusercontent.com/14149777/151560629-8f05a1c3-03a7-4ca2-adbd-98bca4bfdae0.png)
please help me

**Comment by silnarm** [comment, marked as answer]
Don't use the value attribute if you are also using wire:model. Livewire is syncing the state (/value!) of $attachmentFilesName[2] in your component to/from this input, the value attribute you set will just get trounced. If you want that text in the input, you need to assign it to $this->attachmentFilesName[2] in your component.

**Reply by vhdm** [reply]
thanks for your response, but does not work again
![image](https://user-images.githubusercontent.com/14149777/152642914-c72f2a36-389a-48ad-98cc-a64d32735349.png)

**Reply by silnarm** [reply]
Do not use the value attribute if you are also using wire:model!
You are 'wiring' the input to the value of what is under the key $loop->index in the 'attachmentFilesNames' array in your component. Remove the 'value' attribute entirely. With that wire:model attrib, the value shown in the input will be whatever is in that array under that key.
If it does not appear so, double check that array has the values you expect, under the keys you expect. We can't see where that array is being populated/updated, so not much more I can sorry.
Stick a breakpoint in the render method and inspect $this->attachmentFilesName. If you do not have a step debugger setup/working, put this in your render() method:
`Log::info("I am about to render, and my attachmentFilesName array looks like: ", [$this->attachmentFilesName]);`

**Reply by vhdm** [reply]
thanks 👍  ❤️
